### PR TITLE
feat: add toggle switch component

### LINKED ISSUE #18: Toggle Switch Component

### PR DESCRIPTION
**Pull request by tkmct**
Added a toggle switch component.
I'm not very fun of having `id` as a prop, but we need to have id hook like [this](https://github.com/facebook/react/pull/22644) to omit that. For now, I think it's okay to have `id` prop for simplicity.

close #18

### CODE REVIEW

File: packages/kit/src/ToggleSwitch.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import './toggleswitch.css'
+import UIContext from './contexts/interface'
+
+const ToggleSwitch = ({ onChange, checked, id, ...props }) => {
+  const ui = React.useContext(UIContext)
+
+  return (
+    <div className={ui.modeCssClass}>
+      <input
+        id={id}
```
Comment (vimwitch) [inline packages/kit/src/ToggleSwitch.js:11]: The id value should default to a strong random value so the user doesn't have to worry about it. I'd recommend adding the `nanoid` package and using a value generated by that.

Comment (vimwitch) [reply packages/kit/src/ToggleSwitch.js:11]: Actually we can get rid of the `id` prop by using `appearance: none` and removing the label element like [this](https://github.com/tkmct/uikit/compare/component/toggle...vimwitch:switch-refactor?expand=1). Funny enough the check from the checkbox component appears now. Another good reason for #19

**Review by vimwitch — CHANGES_REQUESTED** [review]

**Comment by vimwitch** [comment]
Also what do you think of changing the filename/import to just `Toggle`?

The element looks really good!

File: packages/kit/src/ToggleSwitch.js (hunk 1 above)
Comment (tkmct) [reply packages/kit/src/ToggleSwitch.js:11]: oh, this is interesting approach. I didn't come up with this idea. But it looks simpler and cleaner.
yeah, `appearance: none` is very useful when styling these types of components.
Thanks!!

**Review by vimwitch — APPROVED** [review]

**Pull request merged**